modules/ossm-migrating-complete-remove-maistra-labels.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/done/ossm-migrating-complete-assembly
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-complete-remove-maistra-labels_{context}"]
+= Remove the Maistra labels
+
+After you have removed all {SMProduct} 2 resources, removed the {SMProduct} 2 Operator, and {SMProduct} 2 custom resource definitions (CRDs), you can choose to remove namespace labels created during the migration.
```

Comment:
I wonder if you should have a verb with each phrase, or one for all, to be consistent and clear 

```suggestion
After you have removed all {SMProduct} 2 resources, removed the {SMProduct} 2 Operator, and removed the {SMProduct} 2 custom resource definitions (CRDs), you can choose to remove namespace labels created during the migration.
```
```suggestion
After you have removed all {SMProduct} 2 resources, {SMProduct} 2 Operator, and {SMProduct} 2 custom resource definitions (CRDs), you can choose to remove namespace labels created during the migration.
```